Library/Formula/chicken.rb

```diff
@@ -17,16 +17,16 @@ def install
     ENV.deparallelize
 
     args = %W[
-      PLATFORM=macosx
+      PLATFORM=#{OS.mac? ? "macosx" : "linux"}
       PREFIX=#{prefix}
       C_COMPILER=#{ENV.cc}
       LIBRARIAN=ar
-      POSTINSTALL_PROGRAM=install_name_tool
     ]
 
     # Sometimes chicken detects a 32-bit environment by mistake, causing errors,
     # see https://github.com/Homebrew/homebrew/issues/45648
     args << "ARCH=x86-64" if MacOS.prefer_64_bit?
+    args << "POSTINSTALL_PROGRAM=install_name_tool" if OS.mac?
```

Comment:
Very minor point, but please move this line up to line 25 right after the `]` so it's as close as possible to its original location.